semgrep_output_v1.proto

```diff
@@ -340,27 +340,27 @@ message ParsingStats {
 message ProjectMetadata {
   string semgrep_version = 118812144;
   string repository = 439913047;
-  google.protobuf.Any repo_url = 245057343;
-  google.protobuf.Any branch = 387879587;
-  google.protobuf.Any ci_job_url = 514550082;
-  google.protobuf.Any commit = 200081263;
-  google.protobuf.Any commit_author_email = 458576327;
-  google.protobuf.Any commit_author_name = 534568416;
-  google.protobuf.Any commit_author_username = 247483738;
-  google.protobuf.Any commit_author_image_url = 314681517;
-  google.protobuf.Any commit_title = 23517072;
+  string repo_url = 245057343;
+  string branch = 387879587;
+  string ci_job_url = 514550082;
+  string commit = 200081263;
+  string commit_author_email = 458576327;
+  string commit_author_name = 534568416;
+  string commit_author_username = 247483738;
+  string commit_author_image_url = 314681517;
+  string commit_title = 23517072;
```

Comment:
These should be `optional string`, right?

Reply:
I don't know the semantic of protobuf, nor the jsonschema2protobuf script, but I guess yes. This is what is in the .jsonschema:
```
        "commit_author_email": { "type": [ "string", "null" ] },
        "commit_author_name": { "type": [ "string", "null" ] },
        "commit_author_username": { "type": [ "string", "null" ] },
        "commit_author_image_url": { "type": [ "string", "null" ] },
```